src/python/test/test_extract_expression.py

```diff
@@ -49,18 +53,38 @@ def setUpClass(cls):
         cd(cls.proj_id + ":/")
         cls.general_input_dir = os.path.join(dirname, "expression_test_files/input/")
         cls.general_output_dir = os.path.join(dirname, "expression_test_files/output/")
-        cls.schema = EXTRACT_ASSAY_EXPRESSION_JSON_SCHEMA
+        cls.json_schema = EXTRACT_ASSAY_EXPRESSION_JSON_SCHEMA
+        cls.input_args_schema = EXTRACT_ASSAY_EXPRESSION_INPUT_ARGS_SCHEMA
+        cls.bad_version_dataset = "{}:Extract_Expression/wrong_version_dataset_2".format(cls.proj_id)
+        cls.wrong_type_path_file = "{}:Extract_Expression/wrong_type_file".format(cls.proj_id)
 
         if not os.path.exists(cls.general_output_dir):
             os.makedirs(cls.general_output_dir)
 
     @classmethod
-    def tearDownClass(cls):
-        shutil.rmtree(cls.general_output_dir)
-
+    def path_validation_error_handler(cls, message):
+        raise ValueError(message)
+    
     @classmethod
     def json_error_handler(cls, message):
         raise ValueError(message)
+    
+    @classmethod
+    def tearDownClass(cls):
+        shutil.rmtree(cls.general_output_dir)
+    
+    #
+    # Helper functions used by different types of tests
+    #
+
+    def standard_path_validation_test(self,test_record,expected_error_message):
```

Comment:
Minor: `standard` does not really add any information. Maybe `common` to signify this is shared to all cases?